sass/themes/frost/sr18/sr18.scss

```diff
@@ -11,14 +11,15 @@
 @import "../../../base/variables/_colours.scss";
 
 // Sub theme var overrides
-@import "../../../themes/cr/2017/variables/_colours.scss"; // CR
+@import "../../../themes/sr/2018/variables/_colours.scss"; // SR
+@import "../../../themes/sr/2018/variables/_mixins.scss"; // SR
 @import "../core/variables/_colours.scss"; // Frost
 @import "variables/_colours.scss";
 
 @import "../../../base/variables/_typography.scss";
 
 // Sub theme var overrides
-@import "../../../themes/cr/2017/components/typography/_fonts.scss"; // CR
+@import "../../../themes/cr/2017/components/typography/_fonts.scss"; // SR
```

Comment:
I think this should use sr/2018 fonts, right ?

Reply:
Sorry for confusion, we're sticking with CR fonts for Frost form so I shouldn't update this line of comment to SR ;)